ms/CMakeLists.txt

```diff
@@ -260,6 +261,7 @@ MSSel/MSSelectableTable.h
 MSSel/MSSelection.h
 MSSel/MSSelectionError.h
 MSSel/MSSelectionErrorHandler.h
+MSSel/MSSSpwErrorHanlder.h
```

Comment:
make install should produce an error, normal make or make test should work
maybe there also was a wrongly named file in present in your working copy